sound/soc/sof/intel/hda-dsp.c

```diff
@@ -336,7 +354,17 @@ int hda_dsp_set_power_state(struct snd_sof_dev *sdev,
 	dev_vdbg(bus->dev, "D0I3C updated, register = 0x%x\n",
 		 snd_hdac_chip_readb(bus, VS_D0I3C));
 
-	return 0;
+	/* prevent power gating at D0 */
```

Comment:
can you also align the comments in a separate patch

CIP timeout before D0I3C update
CIP timeout after D0I3C update